pbm/backup/logical.go

```diff
@@ -441,3 +445,32 @@ func getNamespacesSize(ctx context.Context, m *mongo.Client, db, coll string) (m
 	err = eg.Wait()
 	return rv, err
 }
+
+func (b *Backup) checkForTimeseries(ctx context.Context, nss []string) error {
```

Comment:
Probably you wanted to fail fast by checking are we running in sharded cluster: isConfigServer ?